workbench/config/constants.php

```diff
@@ -7,7 +7,7 @@
 //
 //////////////////////////////////////////////////////////////////////////////////////////
 
-$GLOBALS["WORKBENCH_VERSION"] = "43.0.0";
+$GLOBALS["WORKBENCH_VERSION"] = "44.0.0";
```

Comment:
This should not be set. The default version should remain 43 until all orgs have 44 otherwise this will cause customer confusion.